src/libraries/System.Data.Odbc/ref/System.Data.Odbc.ODBC32.cs

```diff
@@ -0,0 +1,20 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Data.Odbc
+{
+    [System.Runtime.CompilerServices.TypeForwardedFrom("System.Data, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089")]
+    public static class ODBC32
+    {
+        [System.Runtime.CompilerServices.TypeForwardedFrom("System.Data, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089")]
+        public enum RETCODE : int
```

Comment:
What is this about? This needs a better comment at least.

Reply:
I added the comment in https://github.com/dotnet/runtime/pull/38968. there is some more info on decision here https://github.com/dotnet/runtime/pull/37944#discussion_r449387420